src/Stryker.Core/Stryker.Core/Mutants/Mutant.cs

```diff
@@ -46,10 +43,6 @@ public class Mutant : IReadOnlyMutant
 
         public string DisplayName => $"{Id}: {Mutation?.DisplayName}";
 
-        public int? Line => Mutation?.OriginalNode?.GetLocation().GetLineSpan().StartLinePosition.Line + 1;
```

Comment:
Ideologically, that's just making core abstractions free of things that are used in some very specific places (a mutant filter and a reporter).

Reply:
However you've now made more consumers responsible for translating the Roslyn Span to user facing Span. If anything I think we should promote Json.Location upwards so we have a proper abstraction for all user facing access of the code span where the span should be 1 instead of 0 indexed. What do you think?

Reply:
Well this might be what we want. F# has its own text span, from the FSharp.Compiler.Test assembly. Now, the difference is quite subtle, basically it's about counting things from 0 vs counting things from 1.

Therefore, we could either generify code based on this type or do some conversions here and there (this is doable). As of now, I am not yet sure which way will produce better code hence I am doing some steps that are supposed to be somewhat beneficial anyway.

Reply:
We will probably need something like a `CSharpMutant` and `FSharpMutant` right? I'm not sure if the PR in its current form contributes towards this goal.